sdk/storage/azure_storage_blob/README.md

```diff
@@ -1,3 +1,140 @@
-# sdk/storage/azure_storage_blob
+# Azure Storage Blobs client library for Rust
 
-TODO TODO TODO
+Azure Blob storage is Microsoft's object storage solution for the cloud. Blob storage is optimized for storing massive amounts of unstructured data, such as text or binary data.
+
+[Source code] | [Package (crates.io)] | [API reference documentation] | [REST API documentation] | [Product documentation]
+
+## Getting started
+
+### Install the package
+
+Install the Azure Storage Blobs client library for Rust with [cargo]:
+
+```sh
+cargo add azure_storage_blob
+```
+
+### Prerequisites
+
+* You must have an [Azure subscription] and an [Azure storage account] to use this package.
+
+
+
+### Create a storage account
+If you wish to create a new storage account, you can use the
```

Comment:
By default does the creator have permissions to read/write blobs? Do they need to grant any permissions to themself?

Reply:
No, they need to add some RBAC roles. @vincenttran-msft please add a section after "Authenticate the client" header called "Permissions" and mention that you may need to setup RBAC roles to access Blob Storage via Microsoft Entra ID. Mostly just link to this page for more info.
https://learn.microsoft.com/azure/storage/blobs/assign-azure-role-data-access?tabs=portal